addons/crm/models/crm_lead.py

```diff
@@ -2094,9 +2084,9 @@ def _creation_subtype(self):
 
     def _creation_message(self):
         self.ensure_one()
-        if self.team_id:
-            return _('A new lead has been created for the team "%(team_name)s".', team_name=self.team_id.display_name)
-        return _('A new lead has been created and is not assigned to any team.')
+        if (not self.partner_id and (intial_values_msg := self._get_initial_values_creation_message_html())):
+            return intial_values_msg
+        return '' if self._creation_subtype() else super()._creation_message()
```

Comment:
So the logic is now:
- If no partner, show "full" detail of values from the template, mostly address/contact stuff
- Then if we have a subtype, as the subtype will print a custom message, don't return anything to avoid duplicates
- Finally, return super to have "Lead Created"

Is that correct?

2 small suggestions:
- Have a quick explanation in commit message
- (Detail) Move towards a if/elif/else structure, would be easier to distinguish the use cases